src/admin/index.js

```diff
@@ -47,7 +47,7 @@ const findTopicPartitions = async (cluster, topic) => {
 module.exports = ({
   logger: rootLogger,
   instrumentationEmitter: rootInstrumentationEmitter,
-  retry = { retries: 5 },
+  retry,
```

Comment:
This change looks a bit suspicious. Am I correct in assuming that `createRetry` will fall back to the default, so that the default here was pointless?

Reply:
Right,

As per documentation for [Admin Client](https://kafka.js.org/docs/admin), Admin Client doesn't provide any default value for itself. It uses the default value provided by the [Client Configuration](https://kafka.js.org/docs/configuration#a-name-retry-a-default-retry).
So, having a default here doesn't make sense.